### ISSUE TITLE
[1846] Add feature to allow players to issue or redeem shares retroactive, even after some actions

### ISSUE THREAD

**Issue by eheiden** [tobymao/18xx#1070]
Technically this breaks the rules but it is an incredibly common thing that happens in person - where they get to train buying step and go "oh, well I will have just have issued one more share" - that it would be very nice to have as a feature, possibly as something people would be able to disable if they want to be super strict with the rules of the game (but I've never played with anyone who hasn't allowed it).

It saves a lot of annoyance during play, having to undo and redo their entire turn.

**Comment by tobymao** [comment]
we've discussed this before in the slack, i don't believe we will do this

**Comment by eheiden** [comment]
You hold the authority, but I will just say that once the game is done and bug free it will likely be the thing that most people will wish they could do with your platform in this game.
